people/ChigoEke.py

```diff
@@ -0,0 +1,5 @@
+def greeting(world):
+    print(f"Hello {world}")
+
+
+greeting("World")
```

Comment:
Looks like we didn't add a final line to the file, could we change that?

# BUT THANK YOU FOR MAKING THIS PULL REQUEST

```suggestion
greeting("World")

```